### ISSUE TITLE
V4.0.15: TypeError: Cannot convert undefined value to object (CSSInterop.StatusBa)

### ISSUE THREAD

**Issue by fendyk** [marklawlor/nativewind#667]
**Describe the bug**
When using V4.0.15 on a Macbook M1 Pro with Sonoma 14.1 and Android emulator, the following error displays right after bundling:

```bash
tailwindcss(native) rebuilding... done
Android Bundling complete 5666ms
 ERROR  TypeError: Cannot convert undefined value to object

This error is located at:
    in CSSInterop.StatusBar (created by Tutorial)
    in RCTView (created by View)
    in View (created by View)
    in View
    in bound createPsuedoClassView (created by Tutorial)
    in Tutorial (created by ContextNavigator)
    in RNCSafeAreaProvider (created by SafeAreaProvider)
    in SafeAreaProvider (created by wrapper)
    in RNGestureHandlerRootView (created by GestureHandlerRootView)
    in GestureHandlerRootView (created by GestureHandlerRootView)
    in GestureHandlerRootView (created by wrapper)
    in wrapper (created by ContextNavigator)
    in ContextNavigator (created by ExpoRoot)
    in ExpoRoot (created by App)
    in App (created by withDevTools(App))
    in withDevTools(App) (created by ErrorOverlay)
    in ErrorToastContainer (created by ErrorOverlay)
    in ErrorOverlay
    in RCTView (created by View)
    in View (created by AppContainer)
    in RCTView (created by View)
    in View (created by AppContainer)
    in AppContainer
    in main(RootComponent), js engine: hermes
 ERROR  TypeError: Cannot convert undefined value to object

This error is located at:
    in CSSInterop.StatusBar (created by Tutorial)
    in RCTView (created by View)
    in View (created by View)
    in View
    in bound createPsuedoClassView (created by Tutorial)
    in Tutorial (created by ContextNavigator)
    in RNCSafeAreaProvider (created by SafeAreaProvider)
    in SafeAreaProvider (created by wrapper)
    in RNGestureHandlerRootView (created by GestureHandlerRootView)
    in GestureHandlerRootView (created by GestureHandlerRootView)
    in GestureHandlerRootView (created by wrapper)
    in wrapper (created by ContextNavigator)
    in ContextNavigator (created by ExpoRoot)
    in ExpoRoot (created by App)
    in App (created by withDevTools(App))
    in withDevTools(App) (created by ErrorOverlay)
    in ErrorToastContainer (created by ErrorOverlay)
    in ErrorOverlay
    in RCTView (created by View)
    in View (created by AppContainer)
    in RCTView (created by View)
    in View (created by AppContainer)
    in AppContainer
    in main(RootComponent), js engine: hermes

```

To Reproduce
- Clone https://github.com/fendyk/expo-router-typescript-nativewind-example
- Run on android
- Wait for bundling complete, should display the error right after

**Expected behavior**
Should not display any errors

**Comment by marklawlor** [comment]
Fix in `4.0.16`

 There is still false positive error with the `verifyInstallation()` that I will need to fix in a future version.

You seem to be testing some styles and are unsure to why they are not working. They are not working due to style specificity. The inline styles of your `themed` `View`/`Text` are overriding the `className` styles. These components are part of the Expo Router example, so you may not have realised that you were using custom `View`/`Text` components. If you do want to use them, your styles will work if you increase the specificity eg change `bg-red-500` to `!bg-red-500`.

There is also an error `IncompatibleNativeValue  display grid` that is caused by the `grid` class name. I'm still working on the wording of this error message :laughing:

**Comment by fendyk** [comment]
> Fix in `4.0.16`
> 
> There is still false positive error with the `verifyInstallation()` that I will need to fix in a future version.
> 
> You seem to be testing some styles and are unsure to why they are not working. They are not working due to style specificity. The inline styles of your `themed` `View`/`Text` are overriding the `className` styles. These components are part of the Expo Router example, so you may not have realised that you were using custom `View`/`Text` components. If you do want to use them, your styles will work if you increase the specificity eg change `bg-red-500` to `!bg-red-500`.
> 
> There is also an error `IncompatibleNativeValue display grid` that is caused by the `grid` class name. I'm still working on the wording of this error message 😆

Thanks for your hard work @marklawlor 🏆
